src/tests/App.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { vi } from 'vitest';
+import { renderWithRedux } from './helpers/renderWith';
+import * as APIModule from '../utils/fetchApi';
+import { News } from '../utils/types';
+import getImageURL from '../utils/getImageURL';
+import { mockNews, mockBreakingNews } from './helpers/mockData';
+import { BASE_IMAGE_URL } from '../components/NewHero/NewHero';
+import App from '../App';
+
+const MOCK_INITIAL_STATE = {
+  news: {
+    status: 'Requisição concluída',
+    news: mockNews,
+    breakingNews: mockBreakingNews,
+    filter: '',
+    quantity: 12,
+    toggleOrientationType: true,
+    activeButton: 'maisRecentes',
+    content: true,
+  },
+};
+
+beforeEach(() => {
+  vi.spyOn(APIModule, 'fetchApi').mockResolvedValue(mockNews as News);
+});
+
+afterEach(() => {
+  vi.clearAllMocks();
+});
+
+const user = userEvent.setup();
+
+describe('Testes da aplicação', () => {
+  test('Verificar o Header', async () => {
+    renderWithRedux(<App />);
+    const header = screen.getByTestId('header');
+    const logoBtn = screen.getByAltText('Logotipo');
```

Comment:
Igualmente aqui pode-se dar preferência ao `getByRole`, especificando o elemento pelo `name`.

```suggestion
    const logoBtn = screen.getByRole('button', {name: "Logotipo"});
```

Como o texto do botão é "Logotipo", ele automaticamente é seu `accessible name`. Você pode conferir como que os accessible names são definidos [nessa documentação](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/)